modules/ROOT/pages/authentication-authorization/ldap-integration.adoc

```diff
@@ -127,7 +127,7 @@ dbms.security.ldap.authorization.group_to_role_mapping=\
 <3> Mapping of two LDAP groups to a Neo4j built-in role.
 <4> Mapping of an LDAP group to a custom-defined role.
 Custom-defined roles, such as `rolename`, must be explicitly created using the `CREATE ROLE rolename` command before they can be used to grant privileges.
-See link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/manage-roles[the Cypher Manual -> Creating roles].
+See xref:authentication-authorization/manage-roles.adoc[the Cypher Manual -> Creating roles].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
See xref:authentication-authorization/manage-roles.adoc[Creating roles].
```